examples/net.yaml

```diff
@@ -0,0 +1,33 @@
+v_nom: 230
+#freq_nom: 50
+ts: .5e-4
+
+components:
+  inv1:
+    #i_nom: 20
+    #i_lim: 30
+    #v_DC: 1000
+    cls: MasterInverter
+    in:
+      u: [i1p1, i1p2, i1p3]    # names of the inputs
+    out:
+      v: [lc1.capacitor1.v, lc1.capacitor2.v, lc1.capacitor3.v]
+      i: [lc1.inductor1.i, lc1.inductor2.i, lc1.inductor3.i]
+    # iref: [0,0,0]
+    # vref: [1,0,0]
+  inv2:
+    cls: SlaveInverter
+    #pll:
+    #  p: 10
+    #  i: 200
```

Comment:
Are these the controller parameters? (Kp & Ki)
If, i (==current) not the best name... shall we keep Kp and Ki?